src/Libero/Sniffs/WhiteSpace/ObjectStaticOperatorSpacingSniff.php

```diff
@@ -11,7 +11,10 @@
 
 final class ObjectStaticOperatorSpacingSniff extends BaseObjectOperatorSpacingSniff
 {
-    public function register()
+    /**
+     * @return array<int>
```

Comment:
Similar to https://github.com/libero/php-coding-standard/pull/40#discussion_r245673354, shouldn't need to re-declare.